### DISCUSSION TITLE
Multicluster communication between clusters without internet

### DISCUSSION THREAD

**Discussion by Noksa** [linkerd/linkerd2#7349] [Q&A]
Hello.

I've tried Linkerd communication between clusters and it is really cool feature.
But I have one question.

I have two clusters in different AWS VPCs but those VPCs are peered. So they work like it is just one network (all pods/nodes can talk to each other without any restrictions).

How can I tell to Linkerd that I don't want to create an internet-faced AWS LB when I use `linkerd multicluster install` command?

I want to create internal AWS LB instead because I can establish connection between clusters without internet.
Or is there proper way to create communication between clusters in local network instead of internet?

Thanks!

**Comment by adleong** [comment]
Hi @Noksa!  The `linkerd multicluster install` command accepts a `--gateway-service-type` flag for changing the service type of the gateway so that you can change the default value of "LoadBalancer".

**Reply by Noksa** [reply]
@adleong 
But it changes only type of service.
I want to make a LoadBalancer type of service but `internal` and not `internet-faced`.

For example if I want to create an internal NLB on AWS I have to add this annotation to a service when I create it:
```
service.beta.kubernetes.io/aws-load-balancer-internal: "true"
```

How can I do it for linkerd's gateway? I don't want to use internet-faced NLB because all pods in VPC peering can reach this NLB in private network.

**Comment by Esardes** [comment]
@Noksa did you ever solve this?

**Reply by Noksa** [reply]
I just use Istio

**Reply by whiskeysierra** [reply]
@Esardes You can set the `gateway.serviceAnnotations` value (either using the multicluster helm chart or use the `--set` flag for the CLI.
